charts/juicefs-s3-gateway/templates/deployment.yaml

```diff
@@ -47,6 +51,14 @@ spec:
                 secretKeyRef:
                   name: juicefs-secret
                   key: secret-key
+            {{- if .Values.envs }}
+            {{ toYaml .Values.envs | nindent 12 }}
```

Comment:
envs are not needed in `auth` container

Reply:
it could be needed when, for example, using google cloud? https://juicefs.com/docs/zh/csi/guide/pv/#mount-pod-extra-files